docs/index.md

```diff
@@ -92,7 +92,7 @@ or as a development dependency for your project:
 $ npm install --save-dev mocha
 ```
 
-> To install Mocha v3.0.0 or newer with `npm`, you will need `npm` v2.14.2 or newer.  Additionally, to run Mocha, you will need Node.js v4 or newer.
+> To install Mocha v3.0.0 or newer with `npm`, you will need `npm` v2.14.2 or newer.  Additionally, to run Mocha, you will need Node.js v6 or newer.
```

Comment:
We should just change this whole thing to "Mocha currently requires Node.js v6 or newer".

we do need to get that netlify issue (#3206) addressed though.

Reply:
I have no experience with Netify. I will test netify as well.